app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitAttendeeNotesApiController.php

```diff
@@ -0,0 +1,228 @@
+
+    /**
+     * @var ISummitRepository
+     */
+    private $summit_repository;
+
+    /**
+     * OAuth2SummitAttendeeNotesApiController constructor.
+     * @param ISummitAttendeeNoteRepository $attendee_notes_repository
+     * @param ISummitAttendeeRepository $attendee_repository
+     * @param ISummitRepository $summit_repository
+     * @param IAttendeeService $attendee_service
+     * @param IResourceServerContext $resource_server_context
+     */
+    public function __construct
+    (
+        ISummitAttendeeNoteRepository $attendee_notes_repository,
+        ISummitAttendeeRepository $attendee_repository,
+        ISummitRepository $summit_repository,
+        IAttendeeService $attendee_service,
+        IResourceServerContext $resource_server_context
+    )
+    {
+        parent::__construct($resource_server_context);
+        $this->attendee_repository = $attendee_repository;
+        $this->summit_repository = $summit_repository;
+        $this->repository = $attendee_notes_repository;
+        $this->attendee_service = $attendee_service;
+    }
+
+    /**
+     * @param $summit_id
+     * @param $attendee_id
+     * @return mixed
+     * @throws \HTTP401UnauthorizedException
+     */
+    public function getAttendeeNotes($summit_id, $attendee_id)
+    {
+        $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->getResourceServerContext())->find($summit_id);
+        if (is_null($summit)) return $this->error404();
+
+        $attendee = $this->attendee_repository->getById(intval($attendee_id));
+        if (is_null($attendee)) return $this->error404();
+
+        return $this->_getAll(
+            function () {
+                return [
+                    'ticket_id' => ['=='],
+                    'author_fullname' => ['=@', '==', '@@'],
+                    'author_email' => ['=@', '==', '@@'],
+                    'created' => ['==', '>=', '<=', '>', '<'],
+                    'edited' => ['==', '>=', '<=', '>', '<'],
+                ];
+            },
+            function () {
+                return [
+                    'ticket_id' => 'sometimes|integer',
+                    'author_fullname' => 'sometimes|string',
+                    'author_email' => 'sometimes|string',
+                    'created' => 'sometimes|required|date_format:U',
+                    'edited' => 'sometimes|required|date_format:U',
+                ];
+            },
+            function () {
+                return [
+                    'id',
+                    'created'
+                ];
+            },
+            function ($filter) {
+                return $filter;
+            },
+            function () {
+                return SerializerRegistry::SerializerType_Private;
+            },
+            null,
+            null,
+            function ($page, $per_page, $filter, $order, $applyExtraFilters) use ($attendee) {
+                return $this->repository->getAllBySummitAttendee
```

Comment:
this is not needed use base method instead
https://github.com/OpenStackweb/summit-api/blob/c9b656034fb16ea5cb7a8b3a0b5af9370b61af5a/app/Repositories/DoctrineRepository.php#L242C21-L242C33
and add filter by owner_id ( attendee id)